text_classifier.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import logging
+import sys, getopt
+import os, inspect
+import numpy as np
+from sklearn.model_selection import train_test_split
+from models.doc2vec_model import doc2VecModel
+from models.classifier_model import classifierModel
+
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO)
+base_file_path = inspect.getframeinfo(inspect.currentframe()).filename
+project_dir_path = os.path.dirname(os.path.abspath(base_file_path))
+data_path = os.path.join(project_dir_path, 'data')
+default_classifier = os.path.join(project_dir_path, 'classifiers','joblib_model.pkl')
```

Comment:
Change `joblib_model.pkl` to `logreg_model`?

Reply:
Sounds more relevant okay (Y)